jp.js

```diff
@@ -60,19 +60,19 @@ generic.login ログイン
 generic.accuracy 命中率
 player.waiting 他のプレーヤーを待っています
 player.died 死亡しました
-player.killed Killed By
+player.killed Killed By // This needs to remain in English due to grammar issues. I want this to be "Killed by {0}."

```

Comment:
need to remove the code comment here, otherwise this happens: https://github.com/mty22/krunker.io-translations/issues/20